catalogue/webapp/server.ts

```diff
@@ -1,22 +1,25 @@
+/* eslint-disable import/first */
 // This needs to be the first module loaded in the application
-require('@weco/common/services/apm/initApm')('catalogue-server');
+import initApm from '@weco/common/services/apm/initApm';
+initApm('catalogue-server');
 
-const Koa = require('koa');
-const Router = require('koa-router');
-const next = require('next');
-const {
+import Koa from 'koa';
+import Router from 'koa-router';
```

Comment:
I don't think these are quite right - I had a headache with this in the identity app. 
- If you inspect the `server`/`router` variables that are created with them, they're `any`s.
- If you switch to `import * as Koa from 'koa'`, then you get errors (because `esModuleInterop` is `true`).
- If you switch to `import Koa = require('koa')`, then it all works _except_ IntelliJ complains that the target module system is not `commonjs`, because it doesn't know about `tsconfig.server.json`.

In the end I gave up with the identity app and they're not properly typed, but it would be really good to get a proper solution for this

Reply:
The server val I have is of type `Koa<Koa.DefaultState, Koa.DefaultContext>` and router `Router<any, {}>`. 

I've installed [@types/koa-router](https://www.npmjs.com/package/@types/koa-router), which make make the difference to the `router`, but not sure why my `server` is different? VSCode?

![Screenshot 2021-10-06 at 13 43 56](https://user-images.githubusercontent.com/31692/136204571-3c6c3a47-8919-427e-92ef-9598fb50c9a4.png)

![Screenshot 2021-10-06 at 13 44 02](https://user-images.githubusercontent.com/31692/136204574-b776ef80-405c-41a7-bc2e-c5735791b8f6.png)

Reply:
Yea, IntelliJ is not playing ball.
![Screenshot 2021-10-06 at 13 47 13](https://user-images.githubusercontent.com/31692/136205059-dfb5165f-38ef-4222-a957-c4a9fbc5d13f.png)

Reply:
Following [this](https://intellij-support.jetbrains.com/hc/en-us/community/posts/115000098070-WebStorm-and-Koa-JS) - I set the `Languages & Frameworks => JavaScript => JavaScript language version: "ECMAScript6+"` and that seemed to have worked

Reply:
Aha, interesting! I had made the mistake of trusting IntelliJ here... Glad it works

Reply:
evidence!
![Screenshot 2021-10-06 at 13 58 59](https://user-images.githubusercontent.com/31692/136207005-7d761bad-0ccd-41a7-bc01-ffa8334e14a0.png)